.changeset/great-coats-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'opticks': patch
+---
+
+:house: Fix GitHub publish action
```

Comment:
In reality this PR would not require a changeset but I'll need to add one now so that the publish action runs